dev-docs/examples/use-prebid-with-appnexus-ad-server.md

```diff
@@ -8,6 +8,7 @@ sidebarType: 1
 about:
 - An example showing how to use Prebid.js with AppNexus Publisher Ad Server
 - See the <a href="https://wiki.appnexus.com/x/PgOXBQ">AppNexus Seller Tag documentation</a> for more information
+- To configure the AppNexus Seller Tag to use SafeFrames, please refer to this <a href="https://wiki.appnexus.com/x/QgOXBQ">page</a> from the documentation on how to enable this setting.
```

Comment:
Change to:
To configure the AppNexus Seller Tag to use SafeFrames, refer to the <a href="https://wiki.appnexus.com/x/QgOXBQ">SafeFrame API Reference</a>.

Reply:
Done